### ISSUE TITLE
Create some tiny wizard / generator / validator for Delphinus json files

### ISSUE THREAD

**Issue by Pawe1** [Memnarch/Delphinus#58]
:bulb: It could be useful because editing files in notepad over and over again is boring and error-prone...

**Comment by Memnarch** [comment]
yup, already commented that on the Synedit-Issue that i should do something like this. At the same time, this project exists:
https://github.com/ms301/Delphinus-its-easy
Seems he started working ot it again, recently.

**Comment by amjadtbssm** [comment]
@Pawe1 is right, A **WYSIWYG** will help the community do the hard and nasty work easily in less time.

**Comment by Pawe1** [comment]
It doesn't compile [missing file `EditorBrowsePath.pas]
@ms301 ? :stuck_out_tongue:

**Comment by Memnarch** [comment]
@file is there, edit the project with a searchpath for it. Bit then again it is quite..broken/unfinished rn :D

**Comment by ms301** [comment]
@Pawe1, The project is under development. :)

**Comment by ms301** [comment]
Unfortunately, I'm a little hung up. In TGrid, I want to use custom editors. But I do not know how to send the changes to "TInstallationFile"

**Comment by Memnarch** [comment]
@ms301 project still in progress :)?
